x/confidentialtransfers/client/cli/query.go

```diff
@@ -178,3 +186,126 @@ func queryAllAccounts(cmd *cobra.Command, args []string) error {
 
 	return nil
 }
+
+// GetCmdQueryTx implements a command to query a tx by it's transaction hash and return it in it's decrypted state.
+func GetCmdQueryTx() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "tx [hash]",
+		Short: "Query the confidential transaction and decrypts it",
+		Long: "Query the confidential transaction by it's tx hash and decrypts it using the private key of the account in --from" +
```

Comment:
Query the confidential transaction by it's tx hash and decrypts it using the private key of the account in --address. For decryption to work, the address should be of a sender, receiver or auditor.

Reply:
Updated description